tests/unit_tests/net.cpp

```diff
@@ -936,6 +936,35 @@ TEST(get_network_address, ipv4subnet)
     EXPECT_STREQ("12.34.0.0/16", address->str().c_str());
 }
 
+TEST(get_network_address, host_and_port)
```

Comment:
I hate bikeshedding, but this really should be `TEST(get_network_address, ipv6)` to be consistent with the other test functions. This is helpful when filtering or identifying tests quickly.

Reply:
Nah that isn't bikeshedding, it does make the tests easier to categorize when they fail.